studio/pages/project/_/[[...routeSlug]].tsx

```diff
@@ -49,9 +49,10 @@ const GenericProjectPage: NextPage = () => {
       }
 
       const slugPath = slug.reduce((a, b) => `${a}/${b}`, '').slice(1)
+
       return [
         `/project/${projectRef}/${slugPath}`,
-        queryString ?? `?${queryString}`,
+        queryString.length > 0 ? `?${queryString}` : undefined,
```

Comment:
There's also same issue on line 46.

This whole logic can be rewritten into (starting on line 36):

```js
const queryString = new URLSearchParams(queryParams as Record<string, string>);

// and then in both places

return [`/project/${projectRef}`, queryString.size ? `?${queryString}` : undefined, hash ?? `#${hash}`]
```

(not sure if `hash` also needs fixing here.

Reply:
ahhh yes! good catch 🙏